modules/aggregation-layers/src/utils/bin-sorter.js

```diff
@@ -74,16 +77,6 @@ export default class BinSorter {
     return [this.sortedBins[lowerIdx].value, this.sortedBins[upperIdx].value];
```

Comment:
sorting is used to quickly return value range by percentile. If the bins are not sorted, here you will have to first sort them before return value range by lower and upper percentile range

Reply:
With this refactor, I am going to use same cpu grid aggregation methods for all the layers (earlier I was using cpu aggregation in `GPUGridAggregator`), what I am doing here is, make sorting optional (by default it is true). Layers other than `CPUGridLayer` and `HexagonLayer`, don't need sorting, they don't support percentile props.

Reply:
as discussed offline, added code to compute sortedBins on demand and save it to the class, removed `sort` flag.